src/components/Task/MassActions/EditTasks/MassActionsEditTasksModal.tsx

```diff
@@ -185,6 +201,11 @@ export const MassActionsEditTasksModal: React.FC<
                     }
                     // None and null are distinct values: null leaves the action unchanged and None changes the action to None
                     preserveNone
+                    required={!!taskPhase}
```

Comment:
It seems it is auto-selecting options when it opens the activity and then you click away.